utensor_cgen/core.py

```diff
@@ -40,6 +41,8 @@ def generate(self, src_fname: str):
     container.template_vars["placeholders"] = []
     container.add_header('"{}"'.format(header_fname))
 
+    opFactory = OperatorFactory()
+
     print("Parsing {}".format(self.pb_file))
     graph_info, layers = parse_pb(self.pb_file)
 
```

Comment:
Why don't you refactor these switch cases, Placeholder and Const, altogether with you `OperatorFactory`?

Reply:
Short answer, I didn't have much time to work on this. They depend on more than just `op_info` so I just added `**kwargs` to hand this in the future.

Reply:
Before I move `Placeholder`, this logic needs to change 
```
container.template_vars["placeholders"].append(out_tname)
header_snippet.template_vars["placeholders"].append(out_tname)
```

Alternatively, I can just replace snippet with the `out_tname`.

Reply:
I see.
I think it's reasonable to do it for now.
OK, I'll merge it after I test it all.
I haven't come up with a good way to automatic test it with a develop board, so I have to do it manually.
It'll take some time

Reply:
Yeah auto testing these things is a nightmare. Ideally this code shouldn't be mbed specific so it should just run in an arbitrary program.